onnxruntime/core/optimizer/gelu_fusion.cc

```diff
@@ -45,7 +45,10 @@ Status GeluFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level, cons
     }
 
     // Check second input is sqrt(2)
-    if (!optimizer_utils::IsInitializerWithExpectedValue(graph, *(div.InputDefs()[1]), static_cast<float>(M_SQRT2), true)) {
+    // Some Bert model uses this approximation of SQRT2 in the Gelu function
```

Comment:
Scott McKay
This comment probably needs updating before adding to github master, as 'our' isn't really meaningful there unless we add the bert model being referred to to that repo.

Sherlock Huang
ok